### DISCUSSION TITLE
Subscribe to MQTT Topic with space character

### DISCUSSION THREAD

**Discussion by JZ-SmartThings** [arendst/Tasmota#10269] [Ideas]
### PROBLEM DESCRIPTION
Apologies for not thinking of this earlier but this is the same as this [issue 10258](https://github.com/arendst/Tasmota/issues/10258) but hoping the same can be done for MQTT Subscribe events. The other side of that is to make it a legal character in %topic%

Quick question: has an MQTT subscribing release of Tasmota been considered? In the same way as IR & Display, etc.

Thanks again, much appreciated!

```lua
  Rules output here:
03:19:37 MQT: stat/TasmotaWallSwitch/RESULT = {"Rule1":"ON","Once":"OFF","StopOnError":"OFF","Length":142,"Free":369,"Rules":"ON mqtt#connected DO Subscribe rcvMQTT, jz/jz#space/a#b ENDON ON Event#rcvMQTT=on DO Power1 1 ENDON ON Event#rcvMQTT=off DO Power1 0 ENDON"}
```

**Comment by JZ-SmartThings** [comment]
@ascillato this was re-classified as an idea rather than issue... Is that because the publish enhancement was simple but subscribe is harder to implement thus less feasible?

**Reply by ascillato** [reply]
Hi,

Feature requests are better to be posted in discussions rather than in issues. Issues are meant for posting bug reports. The issue template is for helping on that.
